### PR TITLE
translate jp new in webaccess

### PR DESCRIPTION
**Pull request by magcho**
I translated the  webaccess

### CODE REVIEW

File: webaccess/src/webaccess_ja_JP.ts [hunk 1]
```diff
@@ -0,0 +1,345 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="fr_FR">
+<context>
+    <name>WebAccess</name>
+    <message>
+        <location filename="webaccess.cpp" line="169"/>
+        <source>Loading project...</source>
+        <translation>プロジェクトを開いています...</translation>
```
Comment (mcallegari) [inline webaccess/src/webaccess_ja_JP.ts]: This and other strings ending with . are warning for Qt Linguist.
However, I have no idea if in Japanese you use punctuation at the end of sentences.
Up to you.

File: webaccess/src/webaccess_ja_JP.ts [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+    </message>
+    <message>
+        <location filename="webaccesssimpledesk.cpp" line="65"/>
+        <source>Next page</source>
+        <translation>次のページ</translation>
+    </message>
+    <message>
+        <location filename="webaccesssimpledesk.cpp" line="68"/>
+        <source>Reset universe</source>
+        <translation>Reset universe</translation>
```
Comment (mcallegari) [inline webaccess/src/webaccess_ja_JP.ts]: Untranslated

File: webaccess/src/webaccess_ja_JP.ts [hunk 3]
```diff
@@ -0,0 +1,345 @@
...
+    </message>
+    <message>
+        <location filename="webaccesssimpledesk.cpp" line="70"/>
+        <source>Universe</source>
+        <translation>Univers</translation>
```
Comment (mcallegari) [inline webaccess/src/webaccess_ja_JP.ts]: Untranslated

File: webaccess/src/webaccess_ja_JP.ts (hunk 1 above)
Comment (magcho) [reply webaccess/src/webaccess_ja_JP.ts]: No problem.
Three dot mean please wait in japanese.
It is not correct grammar. But we are useing it in everyday.

**Comment by mcallegari** [comment]
Thanks a lot ! :+1: 
Would be great if you can have a look also at qlcplus_ja_JP.ts in ui/src

**Pull request merged**